vcd/resource_vcd_vapp_vm_multinetwork_test.go

```diff
@@ -547,7 +547,7 @@ resource "vcd_vapp_vm" "{{.VMName}}" {
 
   network {
     type               = "org"
-    name               = vcd_network_routed.net.name
+    name               = vcd_vapp_org_network.vappAttachedNet.org_network_name
```

Comment:
What is the reason behind all these changes from Routed to vApp Org network?

Reply:
The reason for the failures was that we had the wrong reference for the network name.
Both references in the end result to the same name, but, when we refer to the vapp network, Terraform will wait for the VM to be destroyed before destroying the vapp network. 
When we instead refer to the Org network, Terraform will try destruction in parallel. Sometimes the operation succeeds, and sometimes it fails.

Reply:
Also explained in the [commit](https://github.com/vmware/terraform-provider-vcd/pull/761/commits/bc3df22c798114603be17589616738684303de0d).

Reply:
To add - this actually dates back to when we introduced `vcd_vapp_org_network` resource instead of doing "attachment magic" inside `vcd_vapp_vm`. Back then we tuned our examples and code in many places, however it looks there are still places left.

Reply:
Thank you, this, indeed, reminds me of very similar issues with dependencies I had long time ago.